### PR TITLE
Rebase 1.12.4

### PR DESCRIPTION
**Pull request by soltysh**

### CODE REVIEW

File: pkg/cmd/openshift-sdn/informers.go [hunk 1]
```diff
@@ -13,10 +14,12 @@ import (
 
 // informers is a small bag of data that holds our informers
 type informers struct {
-	KubeClient     kclientset.Interface
+	KubeClient     kubernetes.Interface
 	NetworkClient  networkclient.Interface
 	InternalClient kclientsetinternal.Interface
 
+	// External kubernetes shared informer factory.
+	KubeInformers kinformers.SharedInformerFactory
```
Comment (squeed) [inline pkg/cmd/openshift-sdn/informers.go]: This should just replace `InternalKubeInformers` - nothing uses it.

Comment (squeed) [reply pkg/cmd/openshift-sdn/informers.go]: Er, nevermind. Too many wrappers.

**Comment by soltysh** [comment]
https://zoom.us/j/4799874685

**Comment by soltysh** [comment]
> [fe7bff0](https://github.com/openshift/origin/commit/fe7bff04966a8eb9404a604eb453b3d1998930af) UPSTREAM: 72286: Support KUBE_SSH_BASTION for tunneling SSH

Already present in https://github.com/openshift/kubernetes/tree/origin-4.0-kubernetes-1.12.4, namely https://github.com/openshift/kubernetes/commit/e33b7254d064a7a3edb37e65544fe7c14f514529

> and you really need to pick up an origin version that has [33a7d81](https://github.com/openshift/origin/commit/33a7d812ad0569229bd980d4e136be019a025638)

That's already here, but I'm currently rebasing to pick-up https://github.com/openshift/origin/pull/21832.

**Comment by smarterclayton** [comment]
Master flakes are down significantly, so anything you hit is likely to be you (there is one install flake where the mcd / cmo don't start fully)

**Comment by soltysh** [comment]
/retest

> Master flakes are down significantly, so anything you hit is likely to be you (there is one install flake where the mcd / cmo don't start fully)

I'm gathering data from runs, it's not that straightforward, I'm getting a consistent number of 10-ish failures, but last run from yesterday reached 22, for example. I'm gathering data and will be looking this afternoon.

**Comment by smarterclayton** [comment]
Merging, we have a few flakes but nothing blocking.

**Pull request merged**